suite-common/message-system/src/__fixtures__/messageSystemUtils.ts

```diff
@@ -1225,6 +1225,95 @@ export const validateDeviceCompatibility = [
         },
         result: true,
     },
+    {
+        description: 'validateDeviceCompatibility case 26',
+        deviceConditions: [
+            {
+                model: 'T',
+                firmware: '2',
+                bootloader: '*',
+                firmwareRevision: '123456',
+                variant: '*',
+                vendor: 'trezor.io',
+            },
+            {
+                model: DeviceModelInternal.T2T1,
+                firmware: '2',
+                bootloader: '*',
+                firmwareRevision: '123456',
+                variant: '*',
+                vendor: 'trezor.io',
+            },
+        ],
+        device: {
+            features: {
+                ...getDeviceFeatures({
+                    vendor: 'trezor.io',
+                    internal_model: DeviceModelInternal.T2T1,
+                    major_version: 2,
+                    minor_version: 6,
+                    patch_version: 0,
+                    revision: '123456',
+                }),
+            },
+            firmwareType: 'regular',
+        },
+        result: true,
+    },
+    {
+        description: 'validateDeviceCompatibility case 27',
+        deviceConditions: [
+            {
+                model: '1',
+                firmware: '2',
```

Comment:
possibly false negative? 
```suggestion
                firmware: '1',
```

Reply:
bde981269c3c6b42945670a0bc93cdfe643f3f50